### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ajdavid**
Spelling and grammar corrections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,34 +19,34 @@ This repository contains a few categories of samples:
...
 <a name="quickstarts"></a>
 ## Quickstarts
 
-Here is a set of minial samples for each Cloud Functions trigger types.
+Here is a set of minimal samples for each Cloud Functions trigger types.
 
 ### Quickstart: Uppercaser for [Realtime Database](/quickstarts/uppercase) or [Cloud Firestore](/quickstarts/uppercase-firestore)
 
-This quickstart sample demonstrates using **Cloud Functions** triggered by **Firebase Realtime Database or Cloud Firestore events**. The function uppercase messages written to the datastore.
+This quickstart sample demonstrates using **Cloud Functions** triggered by **Firebase Realtime Database or Cloud Firestore events**. The function sets messages that are written to the datastore, to uppercase.
```
[the remark this answers is not in the document]
Comment (ajdavid) [reply README.md]: Ammended as requested

File: README.md [hunk 2]
```diff
@@ -19,34 +19,34 @@ This repository contains a few categories of samples:
...
 
 ### [HTTPS trigger quickstart: Time Server](/quickstarts/time-server)
 
 This quickstart sample demonstrates using **Cloud Functions** triggered by **HTTPS requests**. The function returns the current server time and allows for date time formatting.
 
 ### [Hosting triggered HTTPS function quickstart: Big Ben](/quickstarts/big-ben)
 
-This quickstart demonstrates using **Cloud Functions** with an HTTPS trigger that's triggered through a Firebase Hosting URL. The function will display a repeated number of "BONG" depending on the hour of the day.
+This quickstart demonstrates using **Cloud Functions** with an HTTPS trigger that's triggered through a Firebase Hosting URL. The function will display a repeated number of "BONG"s, depending on the hour of the day.
```
[the remark this answers is not in the document]
Comment (ajdavid) [reply README.md]: Ammended as requested

File: README.md [hunk 3]
```diff
@@ -19,34 +19,34 @@ This repository contains a few categories of samples:
...
 
 ### [Cloud Storage trigger quickstart: Thumbnail generator](/quickstarts/thumbnails)
 
 This quickstart sample demonstrates using **Cloud Functions** triggered by **Firebase Storage events**. The function generates a thumbnail of uploaded images.
 
 ### [Auth trigger quickstart: Welcome Email](/quickstarts/email-users)
 
-This quickstart sample demonstrates using **Cloud Functions** triggered by **Firebase Auth events**. The functions sends a Welcome email when user accounts are created (or when users signs-in using an Identity Provider for the first time) and sends a Goodbye email when user accounts are deleted.
+This quickstart sample demonstrates using **Cloud Functions** triggered by **Firebase Auth events**. The functions sends a Welcome Email when user accounts are created (or when users sign-in using an Identity Provider for the first time) and sends a Goodbye Email when user accounts are deleted.
```
[the remark this answers is not in the document]
Comment (ajdavid) [reply README.md]: Ammended as requested

File: README.md [hunk 4]
```diff
@@ -67,20 +67,20 @@ JavaScript or TypeScript which requires additional tools.
...
+This requires a build step, included as an npm script, which transpiles the
 source code to a supported version of JavaScript.
 Uses an **HTTPS request**.
 
 ### [Write Cloud Functions using ES2017 code](/es2017-transpile)
 
 This sample shows how you can write your Cloud Functions code using the ES2017
-syntax. This requires to run a preparation step where your code is transpiled to
+syntax. This requires you to run a preparation step where your code is transpiled to
 a supported version of JavaScript.
```
[the remark this answers is not in the document]
Comment (ajdavid) [reply README.md]: Ammended as requested

File: README.md [hunk 5]
```diff
@@ -67,20 +67,20 @@ JavaScript or TypeScript which requires additional tools.
...
 Uses a **Firebase Realtime Database trigger**.
 
 ### [Server-side generated pages w/ Handlebars templating and user sessions](/template-handlebars)
 
-This samples shows how to serve server-side generated HTML pages using the [HandlebarsJs](http://handlebarsjs.com/) templating system and serve User specific content by always passing the Firebase ID token in a `__session` cookie.
+This sample shows how to serve server-side generated HTML pages using the [HandlebarsJs](http://handlebarsjs.com/) templating system and serve User specific content by always passing the Firebase ID token in a `__session` cookie.
```
[the remark this answers is not in the document]
Comment (ajdavid) [reply README.md]: Ammended as requested

File: README.md [hunk 6]
```diff
@@ -134,12 +134,12 @@ Uses a Realtime Database trigger.
...
 
 Keeps track of the number of child nodes of a Firebase Database element allowing clients to filter or order results using the child count.
-This can be useful to keep tack of the number of "likes" or "followers" of a somethings shares through social media.
+This can be useful to keep track of the number of "likes" or "followers" of something shared through social media.
 Uses a Realtime Database trigger.
 
 ### [Limit number of child nodes](/limit-children)
 
-Make sure to keep the number of child nodes below a certain threshold. For instance this can be useful to limit the number of lines of logs or chat history below a given number.
+Makes sure that the number of child nodes stays below a certain threshold. For instance this can be useful to limit the number of lines of logs or chat history below a given number.
```
[the remark this answers is not in the document]
Comment (ajdavid) [reply README.md]: Ammended as requested

File: README.md [hunk 7]
```diff
@@ -188,7 +188,7 @@ Demonstrates hows to integrate Firebase Auth and the Realtime database with Stri
 
 ### [Text moderation](/text-moderation)
 
-How to moderate user input text for bad words. For example this can be used to moderate usernames, chat or forum messages.
+Demonstrates how to moderate user input text for bad words. For example this can be used to moderate usernames, chat or forum messages.
```
[the remark this answers is not in the document]
Comment (ajdavid) [reply README.md]: Ammended as requested

File: README.md [hunk 8]
```diff
@@ -213,8 +213,10 @@ Uses a Realtime Database or Cloud Firestore trigger.
 
 ### [User data cleanup](/user-data-cleanup)
 
-Deletes all associated user data that's on the Realtime database when a user deletes his Firebase account.
+Deletes all associated user data that is in the Realtime database when a user deletes his Firebase account.
```
[the remark this answers is not in the document]
Comment (ajdavid) [reply README.md]: Ammended as requested

File: README.md [hunk 9]
```diff
@@ -243,12 +245,12 @@ Uses an Analytics trigger.
 
 ### [Delete inactive users accounts via cron](/delete-unused-accounts-cron)
 
-Periodically deletes the accounts of users who have not signed in the last month.
+Periodically deletes the accounts of users who have not signed-in in the last month.
```
[the remark this answers is not in the document]
Comment (ajdavid) [reply README.md]: Amended as requested

File: README.md [hunk 10]
```diff
@@ -243,12 +245,12 @@ Uses an Analytics trigger.
...
 Uses an HTTPS trigger.
 
 ### [Developer Motivator](/developer-motivator)
 
-This sample demonstrates how to send a Firebase Cloud Messaging (FCM) notification to the developer device each time your app get a new user or lose one.
+This sample demonstrates how to send a Firebase Cloud Messaging (FCM) notification to the developer device each time your app gets a new user or loses one.
```
[the remark this answers is not in the document]
Comment (ajdavid) [reply README.md]: Amended as requested

**Comment by nicolasgarnier** [comment]
Thanks @ajdavid !

**Pull request merged**